Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView+SwiftUI.swift

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import SwiftUI
+
+@available(iOS 13.0, *)
+/// Protocol of `_ChatChannelListItemView` wrapper for use in SwiftUI.
+public protocol _ChatChannelListItemViewSwiftUIView: View {
+    associatedtype ExtraData: ExtraDataTypes
+    init(dataSource: _ChatChannelListItemView<ExtraData>.ObservedObject<Self>)
+}
+
+@available(iOS 13.0, *)
+extension _ChatChannelListItemView {
+    /// Data source of `_ChatChannelListItemView` represented as `ObservedObject`.
+    public typealias ObservedObject<Content: SwiftUIView> = SwiftUIWrapper<Content> where Content.ExtraData == ExtraData
+
+    /// `_ChatChannelListItemView` represented in SwiftUI.
+    public typealias SwiftUIView = _ChatChannelListItemViewSwiftUIView
+
+    /// SwiftUI wrapper of `_ChatChannelListItemView`.
+    public class SwiftUIWrapper<Content: SwiftUIView>: _ChatChannelListItemView<ExtraData>, ObservableObject
```

Comment:
Let's improve the documentation a bit. It would be great to mention what is this wrapper good for.

Reply:
Documentation updated.